src/emu/debug/debughlp.cpp

```diff
@@ -35,16 +37,29 @@ struct help_item
 
 
 
+
+/***************************************************************************
+    MACROS
+***************************************************************************/
+
+#define HTML_TRY(f) do { if (f < 0) return false; } while (0)
+
+
+
 /***************************************************************************
     LOCAL VARIABLES
 ***************************************************************************/
 
+std::string debug_buffer;
```

Comment:
A global variable like this is a really bad idea.  Also, you shouldn't use a `std::string` as a buffer, it's not efficient.

Reply:
I need it to be a string to use string methods, as well as strreplace(). Will it be enough if I make the buffer local?